common.go

```diff
@@ -250,6 +250,20 @@ func computePayloadHash(version Version, headerHash headerHash, nonce Nonce, cip
 	return sliceToByte64(h)
 }
 
+func computeSigncryptionSignatureInput(headerHash headerHash, nonce Nonce, isFinal bool, chunkPlaintext []byte) []byte {
+	signatureInput := []byte(signatureEncryptedString)
+	signatureInput = append(signatureInput, headerHash[:]...)
+	signatureInput = append(signatureInput, nonce[:]...)
```

Comment:
Hmm. The nonce already includes (part of) the `headerHash` and `isFinal`. Do we still need to mix those in explicitly into `signatureInput`, then?